api/base/renderers.py

```diff
@@ -12,6 +12,11 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         augmented_rendering = re.sub(r'"<esi:include src=\\"(.*?)\\"\/>"', r'<esi:include src="\1"/>', initial_rendering)
         return augmented_rendering
 
+
+class JSONRenderer(JSONAPIRenderer):
```

Comment:
There's no way (that I could find) to specify more than one `media_type` per renderer. Since the stock JSONRenderer was not unescaping the ESI tags I subclassed the JSONAPIRender which subclasses the stock JSONRenderer in order to get the same functionality with the smallest amount of code. Looking back on this, I should have probably made a CustomJSONRenderer that inherits from the stock JSONRenderer, implements the render method and specifies json and application/json and then subclassed to get the JSONAPIRenderer. I'm happy to change it to that to save myself embarrassment and if it will more easily answer your question for future readers.